talkative_robot_2.rb

```diff
@@ -0,0 +1,148 @@
+# puts "Hi, what's your name?"
+# name = gets.chomp
+
+# puts "What is your gender [MF]?" # Experiment with using print instead of puts
+# gender = gets.chomp
+# puts "Ok, how old are you?"
+# age  = gets.chomp.to_i
+def get_name
+	puts "Hi, what's your name?"
+	name = gets.chomp.capitalize
+	return name
+end	
+
+def get_gender
+	puts "Are you male or female? [M/F]"
+	gender = gets.chomp.capitalize
+	return gender
+end	
+
+def get_age
+	puts "And your age?"
+	age = gets.chomp.to_i
+	return age
+end	
+
+user_name = get_name
+user_gender = get_gender
+user_age = get_age
+
+# if age > 55
+# 	greeting = gender == "M" ? "grandfather" : "grandmother"
+# 	puts "Are you a #{greeting}? [Y/N]"
+# 	spawn = gets.chomp
+# 	puts "Aww, congratulations on your big genetic footprint!" if spawn == "Y"
+# elsif age < 18
+# 	greeting = gender == "M" ? "boy" : "girl"
+# 	puts "You're a young #{greeting}! Are you still in school? [Y/N]"
+# 	study = gets.chomp
+# 	puts "I wish I could go to school like you." unless study == "N"
+# else 
+# 	greeting 	= gender == "M" ? "man" : "woman"
+# 	pronoun 	= gender == "M" ? "his" : "her"
+# 	puts "It must be nice to be a #{greeting} who looks younger than #{pronoun} age!"
+# end
+def male?(gender)
+	gender == "M" || gender == "Male"
+end
+
+def under_18?(age)
+	return age < 18
+end
+
+def over_55?(age)
+	return age > 55
+end
+
+if user_age > 55
+	grandparent = male?(user_gender) && over_55?(user_age) ? "grandfather" : "grandmother"
```

Comment:
Don't need the `over_55?(user_age)` -- on the line above, you have `if user_age > 55` so this has to be true.